x-pack/plugins/siem/public/cases/components/case_view/index.tsx

```diff
@@ -244,35 +245,33 @@ export const CaseComponent = React.memo<CaseProps>(
 
     return (
       <>
-        <MyWrapper>
-          <HeaderPage
-            backOptions={{
-              href: getCaseUrl(search),
-              text: i18n.BACK_TO_ALL,
-              dataTestSubj: 'backToCases',
-            }}
-            data-test-subj="case-view-title"
-            titleNode={
-              <EditableTitle
-                disabled={!userCanCrud}
-                isLoading={isLoading && updateKey === 'title'}
-                title={caseData.title}
-                onSubmit={onSubmitTitle}
-              />
-            }
-            title={caseData.title}
-          >
-            <CaseStatus
-              currentExternalIncident={currentExternalIncident}
-              caseData={caseData}
+        <HeaderPage
+          backOptions={{
```

Comment:
could you store `backOptions` and other inline objects as variables to avoid unnecessary re-renders?